overlord/snapstate/backend/export_test.go

```diff
@@ -43,3 +46,15 @@ func MockCommandFromSystemSnap(f func(string, ...string) (*exec.Cmd, error)) (re
 		commandFromSystemSnap = old
 	}
 }
+
+type Interacter interface {
```

Comment:
if the function below is using the exported Interactor from the wrappers package, why do we also introduce one here?

Reply:
Oops! Removed.